src/Extension/ExtensionManager.php

```diff
@@ -260,13 +282,24 @@ protected function setEnabled(array $enabled)
         $this->config->set('extensions_enabled', json_encode($enabled));
     }
 
+    /**
+     * Whether the extension is enabled.
+     *
+     * @param $extension
+     * @return bool
+     */
     public function isEnabled($extension)
     {
         return in_array($extension, $this->getEnabled());
     }
 
+    /**
+     * The extensions path.
+     *
+     * @return string
+     */
     protected function getExtensionsDir()
     {
-        return public_path('extensions');
+        return public_path('vendor');
```

Comment:
While loading the installed.json we also use the hardcoded public path to vendor. IMO that's wrong too. So I just kept it like this. We might need to provide a way to get the vendor_path and extension_path seperately from the ExtensionManager, perhaps in the Application class. Same behavior as Laravel in that sense.

If you don't mind, won't be taking this along for this PR.